channel-connector-archetype/pom.xml

```diff
@@ -12,16 +12,17 @@
   <parent>
     <groupId>com.salesforce.einsteinbot</groupId>
     <artifactId>einstein-bot-channel-connector</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.1-SNAPSHOT</version>
```

Comment:
Not in this PR but eventually we should pull the version out to top level pom and use that in all the submodules. Seems like we have to go and update each submodule separately when we need to update the version.

Reply:
Sure, they are automatically updated by the maven release plugin. Hopefully, it will continue to work if it's pulled to top-level as well.

Reply:
oh ok, never mind then. As long as we don't have to manually update it.